exporter/exporter.go

```diff
@@ -1557,6 +1558,11 @@ func (e *Exporter) connectToRedis() (redis.Conn, error) {
 		uri = "redis://" + uri
 	}
 
+	if len(e.options.PasswordMap) > 0 {
+		log.Debugf("%s pwd : ", e.redisAddr, e.options.PasswordMap[uri])
+		options = append(options, redis.DialPassword(e.options.PasswordMap[uri]))
```

Comment:
Should this only be done when the uri is found? Does it work when using an empty password?

Reply:
Thank you for your suggestion, I have modified the code.

It works when using an empty password.